tlds.go

```diff
@@ -1388,6 +1388,9 @@ var TLDs = []string{
 	`zone`,
 	`zuerich`,
 	`zw`,
+}
+
+var NonASCIITLDs = []string{
```

Comment:
We can't modify this file directly, because it's generated, and because splitting the list would be a backwards-incompatible change.